### PR TITLE
Use absolute paths to locate commandModules

### PR DESCRIPTION
**Pull request by cbush**
- Since moving the commandDir() function, relative paths were broken

### CODE REVIEW

File: src/cli/cli.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import * as Path from "path";
```
Comment (biniona-mongodb) [inline src/cli/cli.ts:1]: Question: Any reason to not just import the `join` method here?

File: src/cli/cli.ts [hunk 2]
```diff
@@ -9,7 +10,7 @@ export async function run(): Promise<void> {
     describe: "add a plugin",
   }).argv;
 
-  const mainArgv = commandDir(yargs.help(), "commandModules").demandCommand();
+  const mainArgv = commandDir(yargs.help(), Path.join(__dirname, "commandModules")).demandCommand();
```
Comment (biniona-mongodb) [inline src/cli/cli.ts:13]: This seems correct. Has this been not working or is this just an improvement?

**Review by biniona-mongodb — APPROVED** [review]
LGTM

File: src/cli/cli.ts (hunk 1 above)
Comment (cbush) [reply src/cli/cli.ts:1]: I like to keep standard library functions namespaced

File: src/cli/cli.ts (hunk 2 above)
Comment (cbush) [reply src/cli/cli.ts:13]: It was using a relative path before, which breaks now that the commandDir file has been moved.

**Pull request merged**